sign_up_form.html

```diff
@@ -0,0 +1,3 @@
+---
+layout: Sign up form
```

Comment:
This will need to be the name of the `html` layout file (without the `.html`). In this case `sign_up_form`.